core/include/mutex.h

```diff
@@ -55,11 +55,14 @@ typedef struct {
 #define MUTEX_INIT_LOCKED { { MUTEX_LOCKED } }
 
 /**
- * @internal
+ * @cond INTERNAL
```

Comment:
I have no idea. I was surprised myself and tried various variants, but couldn't get it to behave as expected.